### DISCUSSION TITLE
gentoo-zh里装的fcitx5如何配置日文罗马音输入？

### DISCUSSION THREAD

**Discussion by guyuming76** [fcitx/fcitx5#609] [Q&A] [answered]
我装了fcitx5-skk,然后在fcitx5-configtool 里添加了 kana.  Ctrl+space切换到jp后，敲键盘可以输入假名。
网上说还可以有罗马音输入法，要怎么配？要装啥包？

**Comment by wengxt** [comment, marked as answer]
首先比较常见的日语输入法是 anthy  / mozc，skk 是比较小众的那一种，skk 在配置里就叫SKK，不叫 Kana。
另外 SKK 的用法是很特别的，想变换汉字要按 shift 的同时再输入。

如果只是想找个常见日语输入法，建议用 anthy 和 mozc，默认配置就是基于罗马音的

**Comment by guyuming76** [comment]
anthy 安装问题：  https://github.com/fcitx/fcitx5/discussions/660
